### DISCUSSION TITLE
How to change color of textField

### DISCUSSION THREAD

**Discussion by DontTellMyMom** [Hopding/pdf-lib#1222] [Q&A]
Hi, 

I am having trouble figuring out how to change the color of a textfield.

I can easily change the text it self, `name.setText(fullName)` but how do i change color?

When using `name.setFontColor(rgb(1, 0, 1));` i get error `[0] TypeError: name.setFontColor is not a function` same happens with [`0]     template.drawText('This text was added with JavaScript!', {`

**Comment by YO-SC** [comment]
I have a lead, but idk where to use em either. I will open a related issue.

https://pdf-lib.js.org/docs/api/interfaces/fieldappearanceoptions#optional-textcolor
